tests/unit/client.py

```diff
@@ -586,6 +586,10 @@ def test_port_revert(self):
 class Test_user:
     """ Tests user related client calls."""
 
+    def test_list_users(self):
+        """Test for getting list of user"""
+        assert C.user.list() == []
```

Comment:
I am getting this error and I have no idea why: 

```
hil/client/user.py:19: in list
    return self.check_response(self.httpClient.request("GET", url))
hil/client/base.py:64: in check_response
    e = json.loads(response.content)
/usr/lib/python2.7/json/__init__.py:339: in loads
    return _default_decoder.decode(s)
/usr/lib/python2.7/json/decoder.py:364: in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

self = <json.decoder.JSONDecoder object at 0x7fd8d4e27450>
s = '<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 3.2 Final//EN">\n<title>Redirecting...</title>\n<h1>Redirecting...</h1>\n<p>Y...target URL: <a href="http://localhost/auth/basic/user/">http://localhost/auth/basic/user/</a>.  If not click the link.'
idx = 0

    def raw_decode(self, s, idx=0):
        """Decode a JSON document from ``s`` (a ``str`` or ``unicode``
            beginning with a JSON document) and return a 2-tuple of the Python
            representation and the index in ``s`` where the document ended.

            This can be used to decode a JSON document from a string that may
            have extraneous data at the end.

            """
        try:
            obj, end = self.scan_once(s, idx)
        except StopIteration:
>           raise ValueError("No JSON object could be decoded")
E           ValueError: No JSON object could be decoded

/usr/lib/python2.7/json/decoder.py:382: ValueError
```
Any thoughts?

Reply:
it probably 404ed. that's what @Izhmash is working on. Let me look at the code to confirm.

Reply:
it's actually a 500